subtests/docker_cli/systemd/p4321-docker-latest.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
```

Comment:
@edsantiago that's my fault.  I suggested having two so we can select between one for docker and another for docker-latest.  But you're right, we don't need two if ``After`` is removed.